src/components/member/ProfileImage.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from 'next/image';
+
+// 기본 이미지 및 사이즈 설정
+const defaultImage = '/svgs/ic_member.svg';
+const size = {
+  lg: 32,
+  sm: 24,
+};
+
+interface ProfileImageProps {
+  userImage?: string;
```

Comment:
이미지 주소는 선택이 아니라 무조건 들어와야 하고 타입이 string|null이어야 할 거 같습니다
```suggestion
  userImage : string | null;
```

Reply:
확인했습니다. 감사합니다!